src/components/common/animations/FaceVerificationErrorSmiley/index.js

```diff
@@ -5,8 +5,12 @@ import AnimationBase from '../Base'
 import animationData from './data.json'
 
 const styles = {
```

Comment:
i see no differences between web ios and android at all, so we could remove `styles` constant and do not use Platform.select

Just pass `style={{ height: '100%' }}` to `Lottie` component in line 27

Reply:
done